src/test/java/com/openlattice/conductor/ConductorBootstrap.java

```diff
@@ -31,29 +31,26 @@
  * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
  */
 public class ConductorBootstrap {
+
     protected static final Indexer INDEXER;
 
     static {
-        INDEXER = new Indexer();
         final var logger = LoggerFactory.getLogger( ConductorBootstrap.class );
+        INDEXER = new Indexer();
+        String[] profiles = new String[3];
+        profiles[0] = "postgres";
```

Comment:
Not important, but can we use predefined constants here, where it's available? For example `ConfigurationConstants.Profiles` or `PostgresPod.PROFILE`

Reply:
very good idea!